components/ecoindex/config/settings.py

```diff
@@ -8,6 +9,7 @@ class Settings(BaseSettings):
     CORS_ALLOWED_ORIGINS: list = ["*"]
     DAILY_LIMIT_PER_HOST: int = 0
     DATABASE_URL: str = "sqlite+aiosqlite:///db.sqlite3"
+    DOCKER_CONTAINER: bool = os.environ.get('DOCKER_CONTAINER', default='False').lower() == 'true'
```

Comment:
```suggestion
    DOCKER_CONTAINER: bool = True
```
No need to read from os environ... Pydantic settings does it by itself :wink:

Reply:
Ah okay ! sorry I am discovering Pydantic..

Reply:
No, don't worry, that's normal if you don't know it :slightly_smiling_face: